### PR TITLE
[#573] Extend README info

### LINKED ISSUE #573: Add extended info in README file
I believe that our README file should present more information involving contributors to participate and explaining the different possibilities that the platform presents.

### PR DESCRIPTION
**Pull request by libremente**
This PR:
* extends the information in the README file
* adds logo and screenshots

Fix:
* Fixes #573

### CODE REVIEW

**Comment by sebbalex** [comment]
@libremente news on this?

**Comment by libremente** [comment]
@sebbalex yes, I need to rebase and add some more info, hopefully I'll be able to do it tomorrow

File: README.md [hunk 1]
```diff
@@ -1,23 +1,92 @@
...
+
+- [Description](#description)
+- [Documentation](#documentation)
+- [Community](#community)
+- [License](#license)
+
+# Description
+
+<div align="center">
+  <img alt="Developers Italia Homepage" src=".github/screenshot.png" />
```
Comment (bfabio) [inline README.md:58]: I'm not sure about the screenshot, it takes away a lot of focus and it'll be outdated pretty quickly by news and other changes.

After all the website is just one click away.

Also, with no screenshot to push the content down we could get away with having no Index, for now.

**Review by bfabio — APPROVED** [review]
That's so much better! :heart: 

I had this branch floating around for a while https://github.com/bfabio/developers.italia.it/tree/readme, but we can merge what's left later if we want those changes as well.

**Review by sebbalex — APPROVED** [review]
:heart: this

File: README.md (hunk 1 above)
Comment (libremente) [reply README.md:58]: Ok makes sense, let's get rid of those things for now.

**Pull request merged**